src/styles/projects-summary.less

```diff
@@ -87,11 +87,88 @@
     a {
       color: @color-pf-white;
       &:hover {
-        color: @link-color;
-        text-decoration: underline;
+        color: @landing-side-bar-link-color-hover;
+        text-decoration: none;
       }
     }
   }
+  .services-view {
+    .services-item {
+      flex: 1 1 auto;
+      margin-bottom: 50px;
+      min-height: 120px;
+      padding: 0;
+      &:focus,
+      &:hover {
+        color: @landing-side-bar-link-color-hover;
+        .services-item-name {
+          color: @landing-side-bar-link-color-hover;
+        }
+      }
+      // Number of recent items displayed based on sidebar width at different viewport sizes
+      @media (max-width: @screen-xxs-max) {
+        &:nth-child(n+3) {
+          display: none; // display first 2
+        }
+      }
+      @media (min-width: @screen-xs-min) {
+        &:nth-child(n+4) {
+          display: none; // display first 3
+        }
+      }
+      @media (min-width: @screen-sm-min) and (max-width: @screen-md-max) {
+        &:nth-child(n+3) {
+          display: none; // display first 2
+        }
+      }
+      @media (min-width: @screen-lg-min) {
+        &:nth-child(n+4) {
+          display: none; // display first 3
+        }
+      }
+      @media (min-width: @screen-xl-min) {
+        &:nth-child(n+3) {
+          display: flex; // display first 4
+        }
+      }
+      .fa-cubes, .icon-wildfly {
+        position: relative;
```

Comment:
Can the position: relative be shared amongst the rules that need it so it isn't duplicated?